test/src/module/command/infoTest.c

```diff
@@ -1308,6 +2359,7 @@ testRun(void)
     {
         StringList *argList = strLstNew();
         strLstAdd(argList, strNewFmt("--repo-path=%s", strZ(repoPath)));
+        strLstAddZ(argList, "--repo1-retention-full=1");  // avoid warning
```

Comment:
Again, this change on line 2362 was not necessary - not sure why it was added and would recommend reverting it.

Reply:
Same as above comment :-)